.github/workflows/tests.yaml

```diff
@@ -18,6 +18,7 @@ on:
     tags:
       - '*'
   workflow_dispatch: {}
+  workflow_call: {}
```

Comment:
I didn't know that these are separate 🤔 

What does this mean? Looking at https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch seems a bit cryptic

BTW: `.github/workflows/release.yaml` above uses `workflow_dispatch` so I'm not sure I understand why that's needed.

Reply:
`workflow_call` makes the workflow callable by other workflows (making it reusable) while `workflow_dispatch` allows triggering the workflow manually (either by a GH UI or CLI).

I added `workflow_call` to `tests.yaml` to make it possible to be called from the `release.yaml` workflow.

Reply:
👍 Thanks for clarification. This makes sense now.